catkit/hardware/newport/NewportMotorController.py

```diff
@@ -9,50 +8,59 @@
 
 """Implementation of the Newport motor controller interface."""
 
+# Import XPS Q8 driver
+try:
+    sys.path.append(os.environ.get('CATKIT_NEWPORT_LIB_PATH'))
```

Comment:
🤦  ``sys.path.append(os.environ.get('CATKIT_NEWPORT_LIB_PATH'))`` can't add ``None`` to ``sys.path``. See https://jira.stsci.edu/browse/CATKIT-89